lib-api/src/main/java/org/ergoplatform/appkit/UnsignedTransaction.java

```diff
@@ -37,4 +37,16 @@ public interface UnsignedTransaction extends Transaction {
      */
     @Nonnull
     List<ErgoToken> getTokensToBurn();
+
+    /**
+     * Json representation of this transaction.
+     */
+    @Nonnull
+    String toJson(boolean prettyPrint);
+
+    /**
+     * Json representation of this transaction.
```

Comment:
please add description of parameters

Reply:
Are you referring to prettyPrint?

Reply:
yes, all parameters, the meaning is not intuitive

Reply:
Tbh I don't understand them, I just followed the implementation for `SignedTransaction`. The parameters were introduced in #73. Maybe @wclaeys can shed some light.

Reply:
hi, surprised you found it problematic :)